src/Summoner/License.hs

```diff
@@ -0,0 +1,50 @@
+module Summoner.License
+       ( License (..)
+       , customizeLicense
+       , licenseNames
+       , githubLicenseQueryNames
+       ) where
+
+import Data.Aeson (FromJSON (..), withObject, (.:))
+import Data.Semigroup ((<>))
+import Data.Text (Text)
+
+import qualified Data.Text as T
+
+----------------------------------------------------------------------------
+-- License
+----------------------------------------------------------------------------
+
+licenseNames :: [Text]
+licenseNames = map fst githubLicenseQueryNames
+
+githubLicenseQueryNames :: [(Text, Text)]
+githubLicenseQueryNames =
+    [ ("MIT",        "mit")
+    , ("BSD2",       "bsd-2-clause")
+    , ("BSD3",       "bsd-3-clause")
+    , ("GPL-2",      "gpl-2.0")
+    , ("GPL-3",      "gpl-3.0")
+    , ("LGPL-2.1",   "lgpl-2.1")
+    , ("LGPL-3",     "lgpl-3.0")
+    , ("AGPL-3",     "agpl-3.0")
+    , ("Apache-2.0", "apache-2.0")
+    , ("MPL-2.0",    "mpl-2.0")
+    ]
+
+newtype License = License { lcnsText :: Text }
```

Comment:
Maybe you could use `License` newtype in `githubLicense*` list and in `customizeLicense` function? With `IsString` instance this should be easier. But I guess this can improve code type safety.